bring-examples/src/test/java/com/bobocode/petros/bring/service/ServiceTest.java

```diff
@@ -0,0 +1,40 @@
+package com.bobocode.petros.bring.service;
+
+import com.bobocode.petros.bring.exception.NoSuchBeanException;
+import com.bobocode.petros.bring.exception.NoUniqueBeanException;
+import com.bobocode.petros.bring.service.impl.MorningServiceImpl;
+import org.junit.jupiter.api.Test;
+
+import static com.bobocode.petros.bring.TestData.CONTEXT;
+import static org.junit.jupiter.api.Assertions.*;
+
+class ServiceTest {
```

Comment:
There are some tests failures, could you please fix the build?

Reply:
It requires aware mechanism and fixes for the component scanner, waiting for merge this 2 tasks

Reply:
got it. I prepared PR where a bug with @Repository and @Service annotations is fixed. Scanner improvements will be provided by @crezione1

Reply:
@Talakh aware mechanism is implemented. You can try adapting it here.